### DISCUSSION TITLE
Is there any way to prevent text in PDF from being selected?

### DISCUSSION THREAD

**Discussion by evgsil** [diegomura/react-pdf#2165] [Q&A] [answered]
I am trying to create big text watermark across entire page. So it overlaps PDF content. Is there any way to prevent text in PDF from being selected? Like `user-select: none;` in CSS

**Comment by jeetiss** [comment, marked as answer]
I have only one idea:

- make an SVG path from the watermark text so it will not be selectable
- place it under the text

**Reply by jeetiss** [reply]
it works: [repl](https://react-pdf-repl.vercel.app/?cp_code=JYWwDg9gTgLgBAbwFBzgFQKYA8YBoVwDKAbgOb6oAKAhjABYVwDijNpGjAasBgO6MARCAGMAriAwA7PAQAywSQGt8AXzgAzKBBBwARAAEoGasJgBaMABN1AeiOTLGI1F0BuJEmERJAZ3i9aJxBqKEUaejgAXgJdAFkANjgAJkSARjgAdjoAVgAbAGY4VML8s2K6eIKikrL8nNyzbKKATjozSrNS1KSzABYipIAJeIAtWPyM5Pi2_IaentTOoqXFpOSyunzqeOa4HbgABkPj3rLcxYuzJLbehvSe7LLqScmjo_TT7LoMqtTtveO72ScGyxHKjWInWoTSab2WJRmIxAPUKC2W6VWkOuT2K1UByzW6XyByaxNhgL6RTB52BhLG8TWGWIvUG2VSnCyqVSSLMzUaFTBzRpqTofxh-KBBxpvWoqSacvxi0KBxucyuZU6nCSYwy_RSDUeSs6Gvl7TgpUKhroMoAHHA7XCPiDpcQhvEmZ1RaksWrsbcyut7sRZvcislrecAxcQQTzUs_WMbZMUhycsHcvNA9Q7Q7jqGDsQ8hqRWcrkVqP1-nCDmX8mCS9kxqkDvd4qnQbNM104P1Q2tM6cnaHQeC26l_at1at0ckwdjspxx3d1Qs2qk7oHZ1y2o3YsU7SkZr6NXXOhmy6utvE7deJT2zpdVqqL30ni9Ffevj9Cf9Eo7zSCYJ1BCULio6SzAfkSIopu0aYlca6yoUuLgYSyQ2kcSQYRKlLeuufZFE22Som2WQdueaxdNm9ofmsBZFhcbRJHM5aVjhlH1juTYZP23pmJyuTxPxFQNHazZrg0lY8jCi4CkkXztEi3SHI0Zh2qU_S7N0bQZCMbgeF4vjwGA1DsIQwAAF4YFEPapM0AB0xHuIZfhwJYwCuZEPYHO42CQLAbkYOo1CiLk8AABQAJRRAAfHA4UEAAPEIYgSNIMUEKgiVsNZPiWRgkS6AAgvEugZagFVwIl3B8JllVwH4ACeuQFQgyD1fVdAYMApB0DAABcejNgcACkuiMB1qAAFaiH4wDqI1ADC3gwFIA16MIa1OONdUddQuQ9ZIACSq0gD4g26Jt0jbRNk2kNQYCDUkBy3ZVKgqLtcDlZNiWYDgMWDBguS5BAcC8NAuSWHAhmrdIiU2H9MAZZ9WXyEocB0EY6iFX1MBgOdNg2KQwD0KIABG9leCANjuRgpDaKIUDUHYximBY1g0x5Yg-Hl3g-DYSSpPE2RlSjFVoIMR2EHAUtwLIR0AHIANJi_DaOKN9lXwzVvDIx11U8LwGjAFgGBQ01LWRG1cCQHlMDAN4F3UGTPgQLkoirbocDvZr9WJSQpBg8Alj0IVSTNBk9nxL0XtdT1fVWyZZn5Wo6jAMDhWSN4GCi5NFUIEVUBM419maNoCV55VCBwC1kikPQg2xLQdD2VoogOOFScYOZVlwDYbkeTA0UANQASor2TeFAD6uBwMA0WRHFwBi6gkX2cEYDheFigYI1C9xRXldZUwK_1TvjVW-fH1H5VMBM746jQCAVsAAZaDAgThWUSTRXf1C-LkT-ZgAAkCBz5wAAFRFF6CoMAWBZ6gNPh1cBUD3J-CQRVWBWBIov2vjfX2R9srNzclbAIq0oDBFCOEOgqd065EKgAYgOMwr2EATLCBJhfBABx7L5GyGoGwBDK6JSYHAP-D8n6FXET4QBq0AAa4VnoHDgZFXON8tbUJIQgMhQQQhhGbrQjOugmEsLgGwkwnCrY8L4QIoRed4ZMDsX7GwjjT6RUing_WNgA5CO1obX28McrfXhilcQa0MqRXcEAA&modules=true)

**Reply by evgsil** [reply]
I'll try this. Thank you!

**Reply by airtonix** [reply]
Doesn't work: 

https://github.com/user-attachments/assets/3a561dd8-86d8-4ccf-9127-4cee36fc9f0a
